packages/core/src/base/cacheManagerKeyv.ts

```diff
@@ -38,46 +38,23 @@ const addCacheHeaders = (resHeaders, path, key, hash, time): void => {
  *
  * @param response The response to calculate the TTL for
  *
- * @returns Time to cache the response in seconds, zero for should not be cached
+ * @returns boolean to cache the response or not
  */
-const getTimeToLiveInSeconds = (response: fetch.Response): number => {
+const isCachable = (response: fetch.Response): boolean => {
   const responseControl = response.headers
     .get("cache-control")
     ?.toLowerCase()
     .trim()
     .split(/\s*,\s*/);
 
-  if (responseControl) {
-    if (
-      responseControl.includes("private") ||
-      responseControl.includes("no-store")
-    ) {
-      return 0;
-    }
-
-    if (responseControl.includes("must-revalidate")) {
-      return Number.MAX_SAFE_INTEGER;
-    }
-
-    const sMaxAgeParts = responseControl
-      .find(value => value.startsWith("s-maxage"))
-      ?.split(/\s*=\s*/);
-    if (sMaxAgeParts?.length > 1 && parseInt(sMaxAgeParts[1]) !== NaN) {
-      return parseInt(sMaxAgeParts[1]);
-    }
-
-    const maxAgeParts = responseControl
-      .find(value => value.startsWith("max-age"))
-      ?.split(/\s*=\s*/);
-    if (maxAgeParts?.length > 1 && parseInt(maxAgeParts[1]) !== NaN) {
-      return parseInt(maxAgeParts[1]);
-    }
+  if (
```

Comment:
You can drop the if and just return the expression. Otherwise this is good. It would also be good to note here that make-fetch-happen assumes its caches are private, but we're going to assume shared.

Reply:
Done